zeek/compiler/driver.cc

```diff
@@ -80,6 +80,11 @@ Driver::Driver(const std::string& argv0, int zeek_version) : spicy::Driver("<Spi
         hilti::configuration().initLocation(argv0);
 
     spicy::Configuration::extendHiltiConfiguration();
+
+    auto& config = spicy::configuration();
+    config.preprocessor_constants["HAVE_ZEEK"] = 1;
```

Comment:
Not really needed right now, but we could prefix this name `SPICY_HAVE_ZEEK`. This would only become relevant should we allow users to set their own preprocessing directives, but might be good to do now that we make these constants available to users.

Reply:
hmm, there's `ZEEK_VERSION` as well, and `SPICY_ZEEK_VERSION` isn't too great. I could see adding an explicit namespace (`Spicy::ZEEK_VERSION`), but it seems overkill right now, and I'm also not very worried about conflicts. Let's just leave as is.